src/pkg/selectors/sharepoint.go

```diff
@@ -268,6 +283,7 @@ const (
 	// types of data identified by SharePoint
 	SharePointWebURL      sharePointCategory = "SharePointWebURL"
 	SharePointSite        sharePointCategory = "SharePointSite"
+	SharePointList        sharePointCategory = "SharePointList"
```

Comment:
You need two layers to complete the scopes a category: a folder and an item.  `List` can act as the folder, so now you need a `ListItem`.

Reply:
Resolved within [8401d93](https://github.com/alcionai/corso/pull/1786/commits/8401d936fa8db1af9e3aa951dbd479cb0e866453)